### DISCUSSION TITLE
jyishit/zhenxun_webui的镜像是没了吗？

### DISCUSSION THREAD

**Discussion by kongxia535** [SinKy-Yan/zhenxunbot-docker#22] [Q&A] [answered]
![d84a7a834152cf2e3d38168a575b4601f0575ac4feed0902f52936bbbc809974QzpcVXNlcnNca29uZ3hpYVxBcHBEYXRhXFJvYW1pbmdcRGluZ1RhbGtcMjY5MTk1NDc1Ml92MlxJbWFnZUZpbGVzXDE2NzI3OTc2ODY2MTBfNEMzOTdDOTktNTVFMy00NTY4LUI5NEMtOTc1REQ1RThCOUUxLnBuZw==](https://user-images.githubusercontent.com/89896964/210480796-47f3cad6-3dbd-4c90-8760-544125a4f5b6.png)

**Comment by SinKy-Yan** [comment, marked as answer]
啊，我把它设为私有了，现在好了应该

**Reply by kongxia535** [reply]
谢谢，看到了
